ui/raidboss/data/triggers/unending_coil_ultimate.js

```diff
@@ -187,96 +199,142 @@
       regexFr: /Des cieux je vais descendre et révérer la lune/,
       infoText: function(data) { return 'Spread => In'; },
       durationSeconds: 6,
-      tts: 'spread then in',
+      tts: {
+        en: 'spread then in',
+        fr: 'Se séparer, puis se rassembler',
+      },
     },
     { id: 'UCU Nael Quote 2',
       regex: /From on high I descend, mine enemies to smite/,
       regexFr: /Du haut des cieux, je vais descendre pour conquérir/,
       infoText: function(data) { return 'Spread => Out'; },
```

Comment:
Did you want to translate these infoTexts too, or should I just duplicate what you put for the tts?

Reply:
Oh I see, you probably skipped these because they were functions.  I think any case where there's a function that just returns a string, we could just drop the function too.